TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -161,6 +161,33 @@ TYPO3:
         'ru': 'Pусский – Russian'
 #        'sv': 'Svenska – Swedish'
 
+      # Aloha presets used in nodeType configuration
+      aloha:
+        presets:
+          'Standard':
```

Comment:
+1 for lowercase, another suggestion would be to call it default instead of standard as it's a more fitting name IMO